test/onnx/test_utility_funs.py

```diff
@@ -602,6 +601,36 @@ def test_error_on_data_parallel(self):
                                     "unwrap model from torch.nn.DataParallel. Try "):
             torch.onnx.export(model, x, f, opset_version=self.opset_version)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_loop_sequence_dim(self):
+        class Module(torch.nn.Module):
+            def forward(self, x, y):
+                l = [x, y]
+                return l
+
+        model = Module()
+
+        x = torch.randn(2, 3)
+
+        # Case 1: varying dimension size.
+        f = io.BytesIO()
+        y = torch.randn(2, 4)
+        torch.onnx.export(torch.jit.script(model), (x, y), f, opset_version=self.opset_version,
```

Comment:
Why is scripting necessary here?

Reply:
added comments

        # Export with scripting to keep output as Sequence type.
        # Tracing unpacks the list.